src/edu/csus/ecs/pc2/core/execute/IExecuteTimerFrame.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+/**
+ * IExecuteTimerFrame interface
+ * This interface should be implemented by classes that use the Executable class.
+ * Typically, this is a Frame, such as AutoJudgeStatusFrame or ExecuteTimerFrame
+ */
+package edu.csus.ecs.pc2.core.execute;
+
+import java.awt.Color;
+
+/**
+ * @author John Buck
+ *
+ */
+public interface IExecuteTimerFrame {
+
+    // sets the frame to it "known" state
```

Comment:
picky picky.  to it's ...
Also "known" is too vague, it this the initial state?

Reply:
It's fixed.

Reply:
confirmed (fixed by removing/replacing comment_